.github/workflows/gradle-build-production.yml

```diff
@@ -19,3 +22,25 @@ jobs:
         run: chmod +x gradlew
       - name: Build with Gradle
         run: ./gradlew assemble
+      - name: Setup gcloud CLI
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          service_account_key: ${{ secrets.RUN_SA_KEY }}
+          project_id: ${{ secrets.RUN_PROJECT }}
+      - name: Build and push image to Google Container Registry
+        run: |-
+          gcloud builds submit \
+            --quiet \
+            --tag "gcr.io/$PROJECT_ID/$SERVICE_NAME:$GITHUB_SHA"
```

Comment:
The way this docker build seems to be happening is very different than dev. What is specifying the server directory for the docker build?

Reply:
Good catch, I grabbed the build step from another project. This has been updated to have Docker build and push the container.